server/liveblog/themes/themes_assets/classic/main.js

```diff
@@ -19,6 +19,21 @@
             });
         }
 
+        function retrieveBlogSettings() {
+            blogsService.get({}, function(blog) {
+                if (blog.title !== vm.blog.title) {
+                    vm.blog.title = blog.title;
+                }
+                if (blog.description !== vm.blog.description) {
+                    vm.blog.description = blog.description;
+                }
+                if (blog.picture_url !== vm.blog.picture_url) {
+                    vm.blog.picture_url = blog.picture_url;
+                    vm.blog.picture_srcset = blog.picture_srcset;
```

Comment:
angular.extend(vm.blog, blog) will do all this checks.
assigning only known properties can cause problems in the future if other properties will be added or will be counted, without knowing what is wrong.

ex: adding a update date for blog in template, you will expect that the update is ( updated ).